docs/source/prototype.models.rst

```diff
@@ -0,0 +1,81 @@
+torchaudio.prototype.models
+===========================
+
+.. py:module:: torchaudio.prototype.models
+
+.. currentmodule:: torchaudio.prototype.models
+
+The models subpackage contains definitions of models and components for addressing common audio tasks.
+
+
+Model Classes
+-------------
+
+Conformer
+~~~~~~~~~
+
+.. autoclass:: Conformer
+
+  .. automethod:: forward
+
+Emformer
+~~~~~~~~
+
+.. autoclass:: Emformer
+
+  .. automethod:: forward
+
+  .. automethod:: infer
+
+RNNT
+~~~~
+
+.. autoclass:: RNNT
+
+  .. automethod:: forward
+
+  .. automethod:: transcribe_streaming
+
+  .. automethod:: transcribe
+
+  .. automethod:: predict
+
+  .. automethod:: join
+
+
+Model Factory Functions
+-----------------------
+
+emformer_rnnt_base
+~~~~~~~~~~~~~~~~~~
+
+.. autofunction:: emformer_rnnt_base
+
+emformer_rnnt_model
```

Comment:
I feel like ``emformer_rnnt_model`` could be above ``emformer_rnnt_base``, as the base model is based on ``emformer_rnnt_model``.